l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -1169,6 +1169,12 @@ def _get_received_date(self, attachment):
         received_date = received_date.date()
         return received_date
 
+    def _get_payment_term(self, partner):
```

Comment:
Non è bloccante, ma mi sembrerebbe più appropriato questo
```suggestion
    def _get_payment_term_id(self, partner):
```